commit d463e1b8ff21fd6c8a3a858bfce9e27469d586af
Author: Andi Shyti <andi.shyti@samsung.com>
Date:   Fri Feb 10 11:20:19 2017 +0900

    spi: s3c64xx: fix inconsistency between binding and driver
    
    commit 379f831a927817c130a62e3ca0082ae685557324 upstream.
    
    Commit a92e7c3d82a1 ("spi: s3c64xx: consider the case when the CS
    line is not connected") introduced an inconsistency between the
    binding, where the disconnected CS line was marked as
    'no-cs-readback', and the driver.
    
    The driver is erroneously checking for that attribute with
    property name of 'broken-cs'.
    
    Check for 'no-cs-readback' in the driver as well.
    
    Fixes: a92e7c3d82a1 ("spi: s3c64xx: consider the case when the CS line is not connected")
    Signed-off-by: Andi Shyti <andi.shyti@samsung.com>
    Reviewed-by: Krzysztof Kozlowski <krzk@kernel.org>
    Signed-off-by: Mark Brown <broonie@kernel.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1228_d463e1b8ff21fd6c8a3a858bfce9e27469d586af.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1228_d463e1b8ff21fd6c8a3a858bfce9e27469d586af.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
